examples/use-cases/Kernel2Vxlan2Kernel&Vfio2Noop/README.md

```diff
@@ -25,6 +25,29 @@ SERVER_MAC=$(mac_create)
 echo Server MAC: ${SERVER_MAC}
 ```
 
+Create kustomization file:
+```bash
+cat > kustomization.yaml <<EOF
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: ns-kernel2vxlan2kernel-vfio2noop
+
+resources:
+- https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/2dbc67a434ce864fe0aca418e578d2fa78e2fcb6/examples/use-cases/Kernel2Vxlan2Kernel&Vfio2Noop/netsvc.yaml
+- https://github.com/networkservicemesh/deployments-k8s/apps/nsc-kernel?ref=2dbc67a434ce864fe0aca418e578d2fa78e2fcb6
+- https://github.com/networkservicemesh/deployments-k8s/apps/nse-kernel?ref=2dbc67a434ce864fe0aca418e578d2fa78e2fcb6
+- https://github.com/networkservicemesh/deployments-k8s/apps/nsc-vfio?ref=2dbc67a434ce864fe0aca418e578d2fa78e2fcb6
+- https://github.com/networkservicemesh/deployments-k8s/apps/nse-vfio?ref=2dbc67a434ce864fe0aca418e578d2fa78e2fcb6
+
+patchesStrategicMerge:
+- https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/2dbc67a434ce864fe0aca418e578d2fa78e2fcb6/examples/use-cases/Kernel2Vxlan2Kernel&Vfio2Noop/patch-nsc.yaml
+- https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/2dbc67a434ce864fe0aca418e578d2fa78e2fcb6/examples/use-cases/Kernel2Vxlan2Kernel&Vfio2Noop/patch-nse.yaml
```

Comment:
Seems like can be a static file.

Also, links can be changed to relative paths.